modules/output/results.py

```diff
@@ -106,6 +111,11 @@ def show(self, currency_symbol: str):
         console.print(table_grid)
 
     def create_trade_info_table(self, justification) -> Table:
+        avg_trade_duration = pandas.Series(self.avg_trade_duration).dt.round('1s')[0].to_pytimedelta() \
+            if self.avg_trade_duration != timedelta(0) else '-'
```

Comment:
something like this:
```suggestion
        avg_trade_duration = timedelta(seconds=round(self.avg_trade_duration.total_seconds())) \
            if self.avg_trade_duration != timedelta(0) else '-'
```

Reply:
Or this? 
```suggestion
        avg_trade_duration = Timedelta(self.avg_trade_duration).round("1s").to_pytimedelta() \
            if self.avg_trade_duration != timedelta(0) else '-'
```

Reply:
Perfect! Went for the first one because it's better readable. Thank you!